dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/acl/AccessStringHelper.java

```diff
@@ -70,6 +70,9 @@ public int getPosition()
 
     public static final String DEFAULT = "--------";
 
+    //This should be used only when creating a default DataElementCategoryOption, DataElementCategory, DataElementCategoryCombo and DataElementCategoryOptionCombo
+    public static final String CATEGORY_DEFAULT = "rwrw----";
+
     public static final String READ = AccessStringHelper.newInstance()
```

Comment:
Currently, only categoryOption has data sharing and not Category and CategoryCombo. 
Saving this  for those objects will lead to the error
 E3011( "Data sharing is not enabled for type `{0}`, but one or more access strings contains data sharing read or write." ),